fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchURITest.java

```diff
@@ -38,8 +38,10 @@ public void testSearchURI_uri() throws Exception {
         assertSearchReturnsSavedResource("uri", "http://hl7.org/fhir/DSTU2");
         assertSearchReturnsSavedResource("uri", "urn:uuid:53fefa32-1111-2222-3333-55ee120877b7");
         
-        // https://github.com/IBM/FHIR/issues/273
         // Matches are supposed to be precise (e.g. case, accent, and escape sensitive), but aren't
```

Comment:
should we remove "but aren't" in the comments?

Reply:
Yes, you are 100% right.  Let me fix